django_select2/forms.py

```diff
@@ -99,8 +99,17 @@ def _get_media(self):
             https://docs.djangoproject.com/en/1.8/topics/forms/media/#media-as-a-dynamic-property
         """
         try:
+            # Values in this dict had been removed from Django as language_code in favor of the keys
+            known_fallback_map = {
```

Comment:
I guess all this can be happening outside the try block, right?
Also I would suggest to fix this also in Django itself... never mind, you already did :)
https://github.com/django/django/commit/e17088a108e604cad23b000a83189fdd02a8a2f9